tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react-jsx"
```

Comment:
After starting project by `npm run start` it changed it automatically from `react` to `react-jsx`

Reply:
It looks like VS Code issue, have a look at the solution suggested here: 

https://stackoverflow.com/questions/64974648/problem-with-visual-studio-code-using-react-jsx-as-jsx-value-with-create-react

Hopefully it helps! :)